libs/render_delegate/utils.cpp

```diff
@@ -381,12 +381,18 @@ bool ConvertPrimvarToBuiltinParameter(
 
 void HdArnoldSetConstantPrimvar(
     AtNode* node, const TfToken& name, const TfToken& role, const VtValue& value, HdArnoldRayFlags* visibility,
-    HdArnoldRayFlags* sidedness, HdArnoldRayFlags* autobumpVisibility, HdArnoldRenderDelegate *renderDelegate)
+    HdArnoldRayFlags* sidedness, HdArnoldRayFlags* autobumpVisibility, 
+    HdArnoldRenderDelegate *renderDelegate, bool skipExisting)
 {
     // Remap primvars:arnold:xyz parameters to xyz parameters on the node.
     if (ConvertPrimvarToBuiltinParameter(node, name, value, visibility, sidedness, autobumpVisibility, renderDelegate)) {
         return;
     }
+    // If this attribute already exists in the node entry parameters list, 
+    // we must skip it #1961
+    if (skipExisting && AiNodeEntryLookUpParameter(AiNodeGetNodeEntry(node), AtString(name.GetText())) != nullptr) {
```

Comment:
I am wondering if we could just test if arnoldAttributeName == str::t_uvs here ? and remove the extra argument skipExisting ?

Reply:
it's because this piece of code in utils is generic and doesn't know if the object is a curves (which expected "uvs"), or if it's a mesh (which would expect "uvlist"). So I expect that at some point we might need the meshes to also use this argument.